src/api-binder.coffee

```diff
@@ -102,9 +102,9 @@ module.exports = class APIBinder
 				@loadBackupFromMigration(retryDelay)
 
 	start: =>
-		@config.getMany([ 'apiEndpoint', 'offlineMode', 'bootstrapRetryDelay' ])
-		.then ({ apiEndpoint, offlineMode, bootstrapRetryDelay }) =>
-			if offlineMode
+		@config.getMany([ 'apiEndpoint', 'unmanaged', 'bootstrapRetryDelay' ])
+		.then ({ apiEndpoint, unmanaged, bootstrapRetryDelay }) =>
+			if unmanaged
 				console.log('Offline Mode is set, skipping API binder initialization')
```

Comment:
We might wanna change the mentions of "offline mode" in the logs (but will approve regardless)